### PR TITLE
Splash screen polish

### PR DESCRIPTION
**Pull request by timglorioso**
Some minor global styling changes were made:
- Font family
- Logo
- Colors

Best effort for non-breaking, ran a visual check of all major areas of the app.

### CODE REVIEW

File: client/src/components/Home.tsx [hunk 1]
```diff
@@ -62,43 +60,55 @@ function Home() {
...
             className={css`
-              width: 100%;
-              max-width: 270px;
-              margin-right: 3rem;
               display: none;
-              transform: scaleX(-1);
-              @media (min-width: ${bp.small}px) {
-                display: inline-block;
+
+              @media (min-width: 768px) {
```
Comment (simonv3) [inline client/src/components/Home.tsx:70]: Would you be down to update the constant.ts file for this?

File: client/src/components/Home.tsx [hunk 2]
```diff
@@ -108,29 +118,80 @@ function Home() {
...
+                className={css`
+                  display: flex;
+                  gap: 16px;
+                `}
+              >
+                <a
+                  href="https://dashboard.mailerlite.com/forms/396303/100612617721087214/share"
+                  target="_blank"
+                  rel="noreferrer"
+                  className={css`
```
Comment (simonv3) [inline client/src/components/Home.tsx:131]: Should we update the `Button` component to match this?

File: client/src/components/common/Button.tsx [hunk 3]
```diff
@@ -77,6 +77,13 @@ const CustomButton = styled.button<Compactable>`
             color: #ddd;
             border-color: #ddd;
           }
+
+          @media (prefers-color-scheme: dark) {
+            color: #F27D98;
```
Comment (simonv3) [inline client/src/components/common/Button.tsx:82]: Should these be stored in `--mi-pink?` but in the `dark` scheme?

**Comment by simonv3** [comment]
This looks great! Made some code-re-use comments but other than that lgtm :)

**Review by simonv3 — APPROVED** [review]

**Comment by timglorioso** [comment]
Since it's only used here, I'm planning on separate "Responsiveness" PRs which will include a more thorough look at breakpoints throughout the app.On Oct 13, 2023, at 20:07, Psi ***@***.***> wrote:﻿
@simonv3 commented on this pull request.



In client/src/components/Home.tsx:
>                display: none;
-              transform: scaleX(-1);
-              @media (min-width: ${bp.small}px) {
-                display: inline-block;
+
+              @media (min-width: 768px) {

Would you be down to update the constant.ts file for this?

**Comment by timglorioso** [comment]
Yeah :) separate PR cool?On Oct 13, 2023, at 20:08, Psi ***@***.***> wrote:﻿
@simonv3 commented on this pull request.



In client/src/components/Home.tsx:
> -              <a ***@***.***">email us</a>.
-            </p>
-            <p>
-              Already have an account? <Link to="/login">Log in</Link>.
-            </p>
+              <div
+                className={css`
+                  display: flex;
+                  gap: 16px;
+                `}
+              >
+                <a
+                  href="https://dashboard.mailerlite.com/forms/396303/100612617721087214/share"
+                  target="_blank"
+                  rel="noreferrer"
+                  className={css`

Should we update the Button component to match this?

**Comment by timglorioso** [comment]
Broken record: separate PR, this one for color scales (design task prereq)On Oct 13, 2023, at 20:10, Psi ***@***.***> wrote:﻿
@simonv3 commented on this pull request.



In client/src/components/common/Button.tsx:
> @@ -77,6 +77,13 @@ const CustomButton = styled.button<Compactable>`
             color: #ddd;
             border-color: #ddd;
           }
+
+          @media (prefers-color-scheme: dark) {
+            color: #F27D98;

Should these be stored in --mi-pink? but in the dark scheme?

**Pull request merged**